src/video_core/renderer_opengl/gl_shader_decompiler.cpp

```diff
@@ -544,13 +562,13 @@ class GLSLGenerator {
                 } else if (cmp_ops.find(op_y) == cmp_ops.end()) {
                     LOG_ERROR(HW_GPU, "Unknown compare mode {:x}", static_cast<int>(op_y));
                 } else if (op_x != op_y) {
-                    shader.AddLine("conditional_code.x = " + src1 + ".x " +
-                                   cmp_ops.find(op_x)->second.first + " " + src2 + ".x;");
-                    shader.AddLine("conditional_code.y = " + src1 + ".y " +
-                                   cmp_ops.find(op_y)->second.first + " " + src2 + ".y;");
+                    shader.AddLine("conditional_code.x = {}.x {} {}.x;", src1,
+                                   cmp_ops.find(op_x)->second.first, src2);
+                    shader.AddLine("conditional_code.y = {}.y {} {}.y", src1,
```

Comment:
```suggestion
                    shader.AddLine("conditional_code.y = {}.y {} {}.y;", src1,
```

This hopefully fixes the issue @tywald saw

Reply:
Yes, applying this change fixes it.

Reply:
Thanks for taking the time to point this out. Amended the PR.